OpenRA.Mods.Common/Projectiles/MeleeAttack.cs

```diff
@@ -0,0 +1,50 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.GameRules;
+using OpenRA.Graphics;
+
+namespace OpenRA.Mods.Common.Projectiles
+{
+	[Desc("Simple invisible direct on target actor fake projectile.")]
+	public class MeleeAttackInfo : IProjectileInfo
```

Comment:
I'd rename this to InstantHitTarget or something along the line because that's what it is and can be used for other things besides melee.